client/rhel/spacewalk-client-tools/src/firstboot-legacy-rhel5/rhn_provide_certificate_gui.py

```diff
@@ -34,7 +34,7 @@ class RhnProvideCertificateWindow(RhnRegisterFirstbootGuiWindow, rhnregGui.Provi
     runPriority=107
     moduleName = _("Provide Certificate")
     windowTitle = moduleName
-    shortMessage = _("Provide a certificate for this Red Hat Satellite server")
+    shortMessage = _("Provide a certificate for this Uyuni server")
```

Comment:
Same observation as above. In this case, it's easy to suggest something downstream-neutral.

```suggestion
    shortMessage = _("Provide a certificate for this Server")
```